.gitignore

```diff
@@ -6,3 +6,5 @@
 /node_modules/
 /dist/
 /lib/
+/nbproject/private/
```

Comment:
This doesn't belong to the itowns's `.gitignore` - gitignore is only populated with folders/files that are by-product or temp files of itowns.

Reply:
(You can add your own ignore stuff in the .gitignore in your home folder)